src/components/LocationDetails.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function LocationDetails({ city, country, errorMessage }) {
+  return errorMessage ? (
+    <h1>{errorMessage}</h1>
+  ) : (
+    <h1 className="location-details">{`${city}, ${country}`}</h1>
```

Comment:
Since there is no CSS for this component, you don't need to add a classname here